DEVELOPING.md

```diff
@@ -4,10 +4,12 @@ As this is a microservice of Courtlistener, tests are designed to be run from a
 to verify that the service works as expected and also works across a docker network.  
 
 ## Quick start
+Generally you can run the following command to build and start your containers
+but if you are on a new AMD64 mac, you may need to comment out the build
+command and use the makefile to build your images.
```

Comment:
Probably better to split this off and say:

 - If you are on x86...
 - If you are arm...